draft-ietf-taps-interface.md

```diff
@@ -652,7 +655,7 @@ the system should fall back to bidirectional transport.
 
 This property specifies whether an application considers it useful to be
 informed in case sent data was retransmitted more often than a certain
-threshold. The recommended default is to have this option.
+threshold. The recommended default is to Prefer this option.
```

Comment:
not Require?

Reply:
No, because it's only available in TCP (I traced it back in the minset => RFC 8303 path). E.g., SCTP can't do that. So this is then up to the transport system to weigh against other preferences... I thought "Prefer" because it seems to make sense to have.

Reply:
It is not clear to me why an SCTP implementation should not be able to gather this kind of information, but I think Prefer is just right. Many stacks will not implement this feature and applications need to be careful in using this signal anyway.

Reply:
SCTP: it's simply not part of the spec. See RFC 8303.

Reply:
hm, i was confused here. question withdrawn.